scripts/dev-shell-build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e -x
+
+cd $(dirname $0)/..
+
+IMAGE_REPO=ranchertest
+
+. ./scripts/version.sh
+
+mkdir -p build
+# getting preloaded images ready
+# creating a special tarball to be used within rke2 dev shell and peer
+docker save ${IMAGE_REPO}/kubernetes:${VERSION} \
```

Comment:
I'd have the dev-shell-build target depend on `build/images/airgap.tar` and then a target for `build/images/airgap.tar` that builds that tar.  So the build will happen once.  Then I'd bind mount build/images to /var/lib/rancher/rke2/agent/images.  I wouldn't copy around.